src/Spec.ts

```diff
@@ -1214,6 +1198,7 @@ ${await utils.readFile(path.join(__dirname, '../js/multipage.js'))}
     const elements = this.doc.body.childNodes;
     const wrapper = this.doc.createElement('div');
     wrapper.id = 'spec-container';
+    wrapper.setAttribute('data-shortname', this.opts.shortname || '');
```

Comment:
```suggestion
    if (this.opts.shortname) {
      wrapper.setAttribute('data-shortname', this.opts.shortname);
    }
```

so we don't have a ton of empty attributes